website/content/docs/connect/config-entries/service-defaults.mdx

```diff
@@ -625,6 +669,26 @@ spec:
         },
       ],
     },
+    {
+      name: 'Destination',
+      type: 'DestinationConfig: <optional>',
+      description: `Controls configuration specific to destinations through terminating-gateway. Added in v1.13.0.`,
+      children: [
+        {
+          name: 'Addresses',
+          type: 'array<string>: []',
+          description: `List of addresses associated with the destination. This can be a hostname or an IP address.
+          Wildcards are not accepted.`,
+          yaml: false,
```

Comment:
We need something in this section since it will be part of the Kube CRDS, correct? Is that what this field is for?